apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -858,7 +861,8 @@ internal DatabaseStatistics GetStatisticsSnapshot()
                         Interlocked.CompareExchange(ref _rwTransactionRequests, 0L, 0L),
                         Interlocked.CompareExchange(ref _rwTransactionRequestsPrewarmed, 0L, 0L),
                         Healthy,
-                        Shutdown);
+                        Shutdown,
+                        _batchCreateSessionRequestTemplate.SessionTemplate.CreatorRole);
```

Comment:
I'd move this to just below `_databaseName` (assuming it's still a separate parameter). That's what it logically belong with.